packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -14,14 +14,17 @@ import { ApplicationShell } from './shell/application-shell';
 import { SHELL_TABBAR_CONTEXT_MENU } from './shell/tab-bars';
 import { AboutDialog } from './about-dialog';
 import * as browser from './browser';
+import { FileSystem } from "@theia/filesystem/lib/common";
```

Comment:
I think making the core depend on a theia-extension is not a good idea (could explain the failed build too).

Because chances are that the core is used in most sub packages, but if you make the core depend on a sub package too, say "hello cycle depedency".

Reply:
moved to editor package